### PR TITLE
added new login experience info to interactive login doc

### PR DESCRIPTION
**Pull request by dbradish-microsoft**
The purpose of this PR is as follows:

1. Add subscription selector information to authenticate-azure-cli-interactively.md
1. Remove the now outdated authenticate-azure-cli-preview.md as this feature has gone to GA and is now in the /*interactively.md document
1. Add the new `enable_broker_on_windows` config setting to azure-cli-configuration.md as this was overlooked in today's WAM PRs
1. Udate the whats-new-overview.md subscription selector H2.

### CODE REVIEW

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 1]
```diff
@@ -13,11 +13,79 @@ ms.custom: devx-track-azurecli
 
 # Sign in interactively with Azure CLI 
 
-Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login) command, which is useful for ad-hoc management tasks and for environments that require manual sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies access for script testing, learning, and on-the-fly management without needing to preconfigure service principals or other noninteractive authentication methods.
+Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login
+with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login)
+command, which is useful for ad-hoc management tasks and for environments that require manual
+sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies
+access for script testing, learning, and on-the-fly management without needing to preconfigure
+service principals or other noninteractive authentication methods.
```
Comment (jiasli) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:21]: Any reason for breaking these lines? They will make `diff`ing difficult if the position of line break character `\n` changes.

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 2]
```diff
@@ -13,11 +13,79 @@ ms.custom: devx-track-azurecli
...
+
+To sign in interactively, use the [az login](/cli/azure/reference-index#az-login) command. Beginning
+with Azure CLI version [2.61.0](./release-notes-azure-cli.md#may-21-2024), Windows systems use Web
+Account Manager (WAM), and Linux and macOS systems use browser-based login by default.
+
+```azurecli-interactive
+az login
+```
+
+## Subscription selector
```
Comment (jiasli) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:37]: This section for **subscription selector** lacks the information on how to disable it.

https://github.com/Azure/azure-cli/pull/28910#issue-2283053031 shows how to disable subscription selector:

```
az config set core.login_experience_v2=off
```

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/authenticate-azure-cli-interactively.md:21]: That is a feature of vsCode and used extensively by both PowerShell and Azure PowerShell writers -- Ctrl D causes every line to not exceed 100 (?) characters.

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md (hunk 2 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/authenticate-azure-cli-interactively.md:37]: fixed in PR https://github.com/MicrosoftDocs/azure-docs-cli/pull/4326/files

**Pull request merged**